.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,84 @@
+name: Feature request
+description: Suggest an idea for this project
+labels: ["enhancement"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request! Please make sure to fill out the entire form below, providing as much context as you can in order to help us triage and track down your request as quickly as possible.
+
+        Before opening a feature request, consider whether the feature should/could be implemented in the [other OpenTelemetry client libraries](https://github.com/open-telemetry/). If so, please [open an issue on opentelemetry-specification](https://github.com/open-telemetry/opentelemetry-specification/issues/new) first.
+
+  - type: dropdown
```

Comment:
As I understand, this does not allow to create uncategorized issues, like https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/1657 (previously it was possible by .github/ISSUE_TEMPLATE/miscellaneous.md)